src/utils/arrayUtils.ts

```diff
@@ -25,3 +25,17 @@ export async function callChunked<Item>(
     await processFn(arrayChunk)
   }
 }
+
```

Comment:
don't forget to update index.ts to export new functions!

Reply:
Ou gosh 😓, sorry, I will add them 🙇